### PR TITLE
Improve README install guide

### PR DESCRIPTION
**Pull request by xyntechx**
## Changes

Separates the recommended and manual install commands for ease of copying.

## Testing

No tests were added as only the README was changed.

## Docs

No docs were updated as only the README was changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,15 @@
 
 ## Install
 
+The **recommended** way to install the latest version of Astro is by running the command below!
```
Comment (sarah11918) [inline README.md:12]:
```suggestion
The **recommended** way to install the latest version of Astro is by running the command below:
```

File: README.md [hunk 2]
```diff
@@ -9,12 +9,15 @@
...
 
 ```bash
-# Recommended!
 npm create astro@latest
+```
+
+However, you can also install Astro **manually** by running this command instead.
```
Comment (sarah11918) [inline README.md:18]:
```suggestion
You can also install Astro **manually** by running this command instead:
```

**Review by sarah11918 — COMMENTED** [review]
Thanks for this, @xyntechx!  We do like keeping our instructions short and sweet, especially when docs has all the details. But, you are absolutely right that when the commands are separate, they are easier to one-click copy.

The only change I'd suggest is that in long, sentence form, we probably don't need the exclamation point anymore!  Now it feels like we're shouting! 😅  With the punctuation changes, Team Docs is officially happy with this PR.

**Comment by xyntechx** [comment]
@sarah11918 thanks for the feedback! I've made and pushed the requested changes to this PR.

**Review by sarah11918 — APPROVED** [review]
Looks Good To Me!  Thank you for this, @xyntechx!  🥳

**Pull request merged**